packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -18,6 +19,16 @@ const propTypes = {
   * Array of links to show for the content of the help button
   */
   helpItems: PropTypes.array,
+  alerts: PropTypes.shape({
```

Comment:
this should probably be distinguished as site alert so its not confused with content alert banners.

Reply:
Would it be simpler if alerts was literally an array of terra-alert components?

Reply:
We don't want {code}handleDismiss{code} to be in {code}alertList{code} array.

Reply:
it would be on each individual terra alert component.

Reply:
I changed it to siteAlert and changed it to an array of terra-alert components.Each alert component will have it's own handleDismiss button.